chapters/03-package-setup.Rmd

```diff
@@ -16,4 +16,26 @@
 
 <!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
+1. Use `create_package()` to create a package called FIXME.  
 
+2. Edit the `DESCRIPTION` file to add:
+
+    * yourself as the author, and
+    * a one sentence description of the package.
+
+3. Add a LICENSE with `use_mit_license()`.
+
+4. Add data to the package
+
+   The raw data and a script to clean the data are provided at: FIXME
+   
+   Following to steps in Section FIXME 
+   to make the FIXME data available as part of the package.
+ 
+You'll know you are finished if:
+
+* Your package passes `check()` 
```

Comment:
Again maybe premature, but I'd strongly advocate this is a part of every final exercise (where changes to the package source occur).

Reply:
Hmm... I get what you mean, but it might be too much, too soon. Sometimes checks give very confusing, hard to identify problems and if we introduce it now it might be a bit overwhelming. Maybe instead of check, it be `install()` or `load_all()`. These run at least very basic checks but aren't as picky as `check()`.

Reply:
Yeah, that's a good idea.  I'll re-write to use `install()` since it would be nice for learners to see at this point they have a functioning/installable package (we can downgrade to `load_all()` if it looks like `install()` will raise issues later).